packages/app-page-builder-elements/src/utils/index.ts

```diff
@@ -49,6 +49,58 @@ export const isPerBreakpointStylesObject = ({
     return false;
 };
 
+// Detect if certain style is a transformed per-breakpoint CSS style object, or a transformed regular CSS properties.
+export const isBreakpointCssStyle = ({
+    breakpoints,
+    styleName
+}: {
+    breakpoints: ThemeBreakpoints;
+    styleName: string;
+}): boolean => {
+    for (const breakpointName in breakpoints) {
+        if (breakpoints[breakpointName] === styleName) {
+            return true;
+        }
+    }
+    return false;
+};
+
+// separate style parts (modifiers) are assigned to different screen sizes one by one
+// this means that "responsive" style properties are assigned
+// in different order compared to theme.breakpoints order
+// it causes not guaranteed CSS order (that causes UI issues)
+// so we need to recreate styles object in proper order
+// (here "order" refers to a for...in cycle order while iterating over the breakpoints)
+// (so we need to recreate styles object adding properties in valid order)
+// see "traversal order" section of for-in loop - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#description
+const recreateStyleObject = ({
```

Comment:
I want to understand the actual issue better.

From what I understood, you're saying that the `Object.keys` sometimes returns the property names in a different order?

![image](https://github.com/webiny/webiny-js/assets/5121148/d052d9d5-b208-4245-9819-2a908535d703)

Reply:
Sorry for taking so long to respond to this....

Can you answer me one thing - which exact issue this changes solves?

I was looking at the Notion doc and thought maybe it was [this one](https://www.notion.so/webiny/Grid-issues-and-a-few-minor-bugs-56bdf708bd574530b872a6ceec441692?pvs=4#519607cb5f0041e08a98c05d86f3977d), but not sure.